src/corell/examples/Cargo.toml

```diff
@@ -0,0 +1,67 @@
+[package]
+name = "gfx-corell-examples"
```

Comment:
I might be missing something here. Why do we have a `Cargo.toml` in the examples folder?